src/main/resources/db/migration/V1_2__prison_supported.sql

```diff
@@ -0,0 +1,12 @@
+DROP TABLE IF EXISTS PRISON_SUPPORTED;
+
+CREATE TABLE PRISON_SUPPORTED
```

Comment:
Are you going to manually populate these migrated ones?  Might be nice to have an endpoint to populates maybe with SYSTEM_USER as protection?

Reply:
Good idea, can add this in next iteration. I was thinking manually populate to begin with.